DataProcessor/DataUnpacker.cpp

```diff
@@ -170,6 +171,10 @@ void DataUnpacker::eng_dash_connection(bool state) {
 }
 
 bool DataUnpacker::checkRestartEnable() {
-    return false; //(!restart_enable ? !mcu_hv_en : false) || driver_eStop || external_eStop || imd_status || door || crash || mcu_check || discharge_enable || restart_enable;
+    return (!restart_enable ? !mcu_hv_en : false) || driver_eStop || external_eStop || imd_status || door || crash || mcu_check || discharge_enable || restart_enable;
+}
+
+void DataUnpacker::enableRestart() {
+    emit sendSignal("mcu_hv_en");
```

Comment:
Just curious, what's the reason for sending just the signal name? Why not just 0 or 1 if we only have one signal? Is it just because the client-side/receiver code is generic on firmware's end? Also, it's just semantics, so don't let it block, but the signal is technically `restart_enable` :neckbeard:. `mcu_hv_en` is what firmware sets after receiving `restart_enable`.

Reply:
A bit of future proofing in case we have more signals